src/Elliptic/Systems/Elasticity/Equations.cpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Elliptic/Systems/Elasticity/Equations.hpp"
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+namespace Elasticity {
+
+template <size_t Dim>
+void primal_fluxes(
+    const gsl::not_null<tnsr::IJ<DataVector, Dim>*> flux_for_displacement,
+    const tnsr::ii<DataVector, Dim>& strain,
+    const ConstitutiveRelations::ConstitutiveRelation<Dim>&
+        constitutive_relation,
+    const tnsr::I<DataVector, Dim>& coordinates) noexcept {
+  const auto stress = constitutive_relation.stress(strain, coordinates);
+  // To set the components of the flux each component of the symmetric stress
+  // tensor is used twice. So the tensor can't be moved in it's entirety. Also,
+  // the stress has lower and the flux upper indices.
```

Comment:
The last sentence might be better located before line 30.

Reply:
Thank you for reviewing it! I tried pushing it, but the checks remained stagnant for a while. I hope the checks pass after rebasing and pushing it again.